validate/validate.go

```diff
@@ -158,7 +158,7 @@ func (v *Validator) CheckPlatform() (msgs []string) {
 		"darwin":    {"386", "amd64", "arm", "arm64"},
 		"dragonfly": {"amd64"},
 		"freebsd":   {"386", "amd64", "arm"},
-		"linux":     {"386", "amd64", "arm", "arm64", "ppc64", "ppc64le", "mips64", "mips64le", "s390x"},
+		"linux":     {"386", "amd64", "arm", "arm64", "mips", "mipsle", "mips64", "mips64le", "ppc64", "ppc64le", "s390x"},
```

Comment:
Also, the fact that Go uses `mips64le` and libseccomp uses `SCMP_ARCH_MIPSEL64` is unfortunate.  Do we want to do anything about supporting either permutation for folks who assume that one value (e.g. `mips64le`) will work for both `--arch` and `--seccomp-arch`?